src/pages/reader.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="h-screen">
```

Comment:
`h-screen`(100vh) behavior is strange in iOS, should hack Tailwind CSS to use 100dvh with 100vh as fallback

https://github.com/likecoin/liker-land/assets/17264716/194ef198-43a0-4c78-ba0a-6412c692b9e8

https://stackoverflow.com/questions/75079019/tailwind-css-fallback-for-new-screen-length-types-such-as-lvh-svh